realm/realm-library/src/main/cpp/jni_util/jni_utils.hpp

```diff
@@ -31,12 +35,16 @@ class JniUtils {
 
     // Call this only once in JNI_OnLoad.
     static void initialize(JavaVM* vm, jint vm_version) noexcept;
+    // Call this in JNI_OnUnload.
+    static void release();
```

Comment:
`releaseGlobalRefs()` would probably be a better name. Right now it is very generic?